src/overlays/actors/ovl_En_Si/z_en_si.c

```diff
@@ -80,7 +80,7 @@ s32 func_80AFB748(EnSi* this, GlobalContext* globalCtx) {
 void func_80AFB768(EnSi* this, GlobalContext* globalCtx) {
     Player* player = GET_PLAYER(globalCtx);
 
-    if (CHECK_FLAG_ALL(this->actor.flags, ACTOR_FLAG_13)) {
+    if (CHECK_FLAG_ALL(this->actor.flags, ACTOR_FLAG_HOOK_ATTACHED)) {
```

Comment:
imo en_si doesnt really care about that, the hack is the boomerang using it

Reply:
I feel like someone reading ensi code would appreciate knowing this also handles boomerang. I think it's worth documenting since the code works with more than hookshot (kind of unintendedly so?)